Cargo.toml

```diff
@@ -165,28 +165,28 @@ k8-diff = { version = "0.1.2" }
 trybuild = { branch = "check_option", git = "https://github.com/infinyon/trybuild" }
 
 # Internal fluvio dependencies
-fluvio = { version = "0.22.0", path = "crates/fluvio" }
```

Comment:
Removing versions doesn't cause problems with crate publishing anymore?

Reply:
Good catch, it seems like it's still an issue: https://github.com/rust-lang/cargo/issues/11133

The odd part is that it seems like the version at the workspace Cargo.toml doesnt seem to be used for publish - the crate version for fluvio was published at 0.22.1 last stable release.  I wonder if we can fill in version =  "0.0.0"?

Reply:
Added a commit to restore version property on those deps